src/libexpr/value/print.cc

```diff
@@ -0,0 +1,26 @@
+#include "value/print.hh"
```

Comment:
It seems a bit arbitrary to introduce a "value" subdirectory, when nothing else value-related is there...

Reply:
We already have a value subdirectory. I previously made `value/context.{hh,cc}` on analogy with `primops/context.cc`. String contexts are part of string values.

Reply:
While `value/` was not a terrible choice, generalizing the file to allow printing all of the language is more useful. I've renamed it to just `print.{cc,hh}` in the `libexpr` top level.